### PR TITLE
Add pull-request guidelines to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jsumners**
This PR updates the CONTRIBUTING.md document to set for some guidelines for managing pull requests. @fastify/fastify please review it and give your feedback. Should it be merged? Are there things that should be changed? Is this unnecessary?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ There are a few basic ground-rules for contributors:
 1. **External API changes and significant modifications** ought to be subject to an **internal pull-request** to solicit feedback from other contributors.
 1. Internal pull-requests to solicit feedback are *encouraged* for any other non-trivial contribution but left to the discretion of the contributor.
 1. Contributors should attempt to adhere to the prevailing code-style.
+1. At least two contributors, or one core member, must approve pull-requests prior to merging.
+1. All integrated CI services must be green, unless an agreement has been reached in the conversation about the exception, before a pull-request can be merged.
+1. Large pull-requests should be merged by a core member. Small pull-requests, e.g. documentation typo corrections, may be merged by the pull-request author if they have commit access.
```
Comment (mcollina) [inline CONTRIBUTING.md]: I would rephrase this as "breaking changes or significant new feaures". I would remove the "pull-request author".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ There are a few basic ground-rules for contributors:
 1. **External API changes and significant modifications** ought to be subject to an **internal pull-request** to solicit feedback from other contributors.
 1. Internal pull-requests to solicit feedback are *encouraged* for any other non-trivial contribution but left to the discretion of the contributor.
 1. Contributors should attempt to adhere to the prevailing code-style.
+1. At least two contributors, or one core member, must approve pull-requests prior to merging.
```
Comment (StarpTech) [inline CONTRIBUTING.md]: I wouldn't make differences between core and community members. It should based on the project experience and of the knowledge of the developer. I would just say: At least two members are required to approve a PR. For none code changes e.g documentation it's enough to have one approval of a member. Significant architectural changes should be announced to the whole team and need at least one approval of a team member who has same understanding of the change like you.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -13,6 +13,9 @@ There are a few basic ground-rules for contributors:
 1. **External API changes and significant modifications** ought to be subject to an **internal pull-request** to solicit feedback from other contributors.
 1. Internal pull-requests to solicit feedback are *encouraged* for any other non-trivial contribution but left to the discretion of the contributor.
 1. Contributors should attempt to adhere to the prevailing code-style.
```
Comment (StarpTech) [inline CONTRIBUTING.md]: A badge is not always the best to define code-style. Link to docs or standardjs ?

Comment (mcollina) [reply CONTRIBUTING.md]: we are using standardjs

Comment (StarpTech) [reply CONTRIBUTING.md]: I mean we should write it down.

Comment (mcollina) [reply CONTRIBUTING.md]: Agreed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jsumners) [reply CONTRIBUTING.md]: Updated.

File: CONTRIBUTING.md (hunk 3 above)
Comment (jsumners) [reply CONTRIBUTING.md]: That's outside of the scope of this PR.

File: CONTRIBUTING.md (hunk 2 above)
Comment (mcollina) [reply CONTRIBUTING.md]: I think we should live with this and try out how it goes. It is significantly different than what we do in node core, and I'm 👍  to experiment!

File: CONTRIBUTING.md (hunk 3 above)
Comment (delvedor) [reply CONTRIBUTING.md]: > That's outside of the scope of this PR.

Agree.

Don't forget that if you press on the "code style standard" badge in our README.md you will be redirected to the standardjs website :)
